core/src/trezor/ui/layouts/common.py

```diff
@@ -29,3 +21,11 @@ async def interact(
     workflow.close_others()
     await ctx.call(ButtonRequest(code=brcode), ButtonAck)
     return await ctx.wait(layout)
+
+
+async def not_cancelled(a: Awaitable[None]) -> bool:
```

Comment:
ISTM all uses of `not_cancelled` would be superseded by passing a custom exception instance to `show_error` or `confirm_action`, correct?

Reply:
Yeah, good idea:+1:. Fixed in 3c39b4ee1b842dbf9baf26e2491f4697009a1b45.